client/my-sites/domains/domain-search/site-redirect-step.jsx

```diff
@@ -1,90 +1,101 @@
+import upgradesActions from 'lib/upgrades/actions';
+import { recordGoogleEvent } from 'state/analytics/actions';
+
+class SiteRedirectStep extends React.Component {
+	static propTypes = {
 		cart: React.PropTypes.object.isRequired,
 		products: React.PropTypes.object.isRequired,
-		selectedSite: React.PropTypes.object.isRequired,
-	},
+		selectedSite: React.PropTypes.object.isRequired
+	};
 
-	getInitialState: function() {
-		return { searchQuery: '' };
-	},
+	state = {
+		searchQuery: ''
+	};
 
-	render: function() {
-		var price = this.props.products.offsite_redirect ? this.props.products.offsite_redirect.cost_display : null;
+	render() {
+		const { translate } = this.props;
+		const price = this.props.products.offsite_redirect ? this.props.products.offsite_redirect.cost_display : null;
 
 		return (
-			<div className="site-redirect-step">
-				<form className="site-redirect-step__form card" onSubmit={ this.handleFormSubmit }>
-
-					<div className="site-redirect-step__domain-description">
-						<p>
-							{ this.translate( 'Redirect {{strong}}%(domain)s{{/strong}} to this domain', {
-								components: { strong: <strong /> },
-								args: { domain: this.props.selectedSite.slug }
-							} ) }
-						</p>
-					</div>
-
-					<DomainProductPrice
-						price={ price }
-						requiresPlan={ false } />
-
-					<fieldset>
-						<input
-							className="site-redirect-step__external-domain"
-							type="text"
-							value={ this.state.searchQuery }
-							placeholder={ this.translate( 'Enter a domain', { textOnly: true } ) }
-							onChange={ this.setSearchQuery }
-							onClick={ this.recordInputFocus } />
-						<button className="site-redirect-step__go button is-primary"
-								onClick={ this.recordGoButtonClick }>
-							{ this.translate( 'Go', {
-								context: 'Upgrades: Label for adding Site Redirect'
-							} ) }
-						</button>
-					</fieldset>
-				</form>
+			<div>
+				<div className="domain-search__site-redirect-step">
+					<Card>
+						<form onSubmit={ this.handleFormSubmit }>
+
+							<div className="domain-search__site-redirect-step-description">
+								<p>
+									{ translate( 'Redirect {{strong}}%(domain)s{{/strong}} to this domain', {
+										components: { strong: <strong /> },
+										args: { domain: this.props.selectedSite.slug }
+									} ) }
+								</p>
+							</div>
+
+							<DomainProductPrice
+								price={ price }
+								requiresPlan={ false } />
+
+							<fieldset>
+								<FormTextInputWithAffixes
+									type="text"
+									value={ this.state.searchQuery }
+									prefix="http://"
+									placeholder={ translate( 'Enter a domain', { textOnly: true } ) }
+									onChange={ this.setSearchQuery }
+									onClick={ this.recordInputFocus } />
+							</fieldset>
+							<FormFooter>
+								<FormButton
+									onClick={ this.recordGoButtonClick }>
+									{ translate( 'Add Site Redirect', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?